plugins/@grouparoo/vero/src/lib/export/listMethods.ts

```diff
@@ -0,0 +1,18 @@
+import { VeroClient } from "../client";
+
+export async function handleGroupChanges(
+  client: VeroClient,
+  oldGroups: Array<string>,
+  newGroups: Array<string>,
+  newRecordProperties: { [p: string]: any }
+) {
+  const groupsToRemove = oldGroups.filter(
+    (groupName) => !newGroups.includes(groupName)
+  );
+
+  await client.updateUserTags(
+    newRecordProperties["id"],
+    newGroups.map((groupName) => `In Group: ${groupName}`),
```

Comment:
We don't need to prepend`"In Group:"`. They can set it to the tag name they want in the config